public/index.html

```diff
@@ -1,46 +1,45 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-US">
```

Comment:
Is there a special reason for using US here? As far as I know it is best to keep the tags as short as possible https://www.w3.org/International/articles/language-tags/